bin/ds

```diff
@@ -142,6 +142,24 @@ function build {
   echo "Creating symlinks"
   symlinks
 
+  echo "Enabling git hooks"
+  if [[ -d "$BASE_PATH/.git/hooks/" ]]; then
+    # Pre-commit.
+
+    HOOK_PRE_COMMIT=$BASE_PATH/.git/hooks/pre-commit
+    if [[ ! -f $HOOK_PRE_COMMIT ]]; then
+      if [[ -L $HOOK_PRE_COMMIT ]]; then
+        # Symlink is broken.
+        rm $HOOK_PRE_COMMIT
```

Comment:
why if it's a not a file and symlink why is it removing it?
can you add documentation to say what's going on here?